Library/Formula/dynamips.rb

```diff
@@ -1,21 +1,25 @@
-require 'formula'
-
 class Dynamips < Formula
-  homepage 'http://www.gns3.net/dynamips/'
-  url 'https://downloads.sourceforge.net/project/gns-3/Dynamips/0.2.10/dynamips-0.2.10-source.zip'
-  sha1 '6b8fc56e21db2fabfa684813b3fa56b59c6fce0a'
+  homepage "http://www.gns3.net/dynamips/"
+  url "https://github.com/GNS3/dynamips/archive/v0.2.14.tar.gz"
+  sha1 "1f0b62d19586365246a957b4eb4dab0cdbd657ad"
 
-  depends_on 'libelf'
+  depends_on "libelf"
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
```

Comment:
@mikemcquaid I got this failure from `cmake`:

```
CMake Error at cmake/dependencies.cmake:27 (message):
  Not a GNU C compiler.  The source and build system assumes gcc so it might
  not compile.  Invoke cmake with -DANY_COMPILER=1 to skip this check.
Call Stack (most recent call first):
  CMakeLists.txt:32 (include)
```

I then added this `gcc` dependency but it failed again. I added `-DANY_COMPILER=1` and the compilation was fine, both with and without this dependency, so I guess it’s not useful. I just updated the PR to remove it.